csfieldguide/chapters/content/en/data-representation/sections/text.md

```diff
@@ -110,7 +110,7 @@ Nowadays ASCII has been supplanted by a code called "UTF-8", which happens to be
 
 Have a go at the following ASCII exercises:
 
-- How would you represent "science" in ASCII?
+- How would you represent "science" in ASCII (ignore the `"` marks)?
 - How would you represent "Wellington" in ASCII? (note that it starts with an upper case "W")
 - How would you represent "358" in ASCII (it is three characters, even though it looks like a number)
```

Comment:
This needs a question mark and technically the one below it as well. Also maybe we should put the part inside parentheses after the question mark on the first bullet point to be consistent.